tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/preprocessor_1k.cc

```diff
@@ -0,0 +1,57 @@
+/* This file is a modification of the Tensorflow Micro Lite file preprocessor.cc
+ * We have retained the original copyright and header information, in
+ * accordance with the Apache 2.0 license terms.
+ */
+
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/experimental/micro/examples/micro_speech/CMSIS/sin_1k.h"
+#include "tensorflow/lite/c/c_api_internal.h"
+#include "tensorflow/lite/experimental/micro/micro_error_reporter.h"
+#include "tensorflow/lite/experimental/micro/testing/micro_test.h"
+
+extern "C" {
+#include "tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/system_apollo3.h"
+#include "tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/apollo3.h"
+}
+
+#define output_data_size 43
+int count;
+
+extern TfLiteStatus Preprocess(tflite::ErrorReporter* error_reporter,
+                        const int16_t* input, int input_size, int output_size,
+                        uint8_t* output);
+
+TF_LITE_MICRO_TESTS_BEGIN
+CoreDebug->DEMCR |= CoreDebug_DEMCR_TRCENA_Msk;
+//DWT->LAR = 0xC5ACCE55; 
```

Comment:
Maybe remove this commented-out code, and add a comment about what the core debug flag is all about?